java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgFrame.java

```diff
@@ -1580,8 +1578,8 @@ public void enableButtons(boolean stat) {
         }
         writeChangesButton.setEnabled(stat);
         writeAllButton.setEnabled(stat);
-        if (modePane != null) {
-            modePane.setEnabled(stat);
+        if (getModePane() != null) {
+            getModePane().setEnabled(stat);
```

Comment:
No need to change. But it might be good to have:
```
JPanel modePane = getModePane();
if (modePane != null) {
    modePane.setEnabled(stat);
```

The reason is that NPE analysis don't know that `getModePane()` will not change between calls.